### PR TITLE
konvoy: Update docs for konvoy release v1.5.0-beta.8

### PR DESCRIPTION
**Pull request by dkoshkin**
Release notes and other minor changes for Konvoy `v1.5.0-beta.8`

### CODE REVIEW

File: pages/ksphere/konvoy/1.5.0-beta/gpu/index.md [hunk 1]
```diff
@@ -33,13 +33,7 @@ The following components provide Nvidia GPU support on Konvoy:
...
 1.  NVIDIA GPU with Fermie architecture version 2.1 or greater.
 
-1.  Ensure the kernel-headers version is identical to the kernel version on each node. To verify this use the following script:
-
-    * Ubuntu
-
-    ```bash
-    KERNEL_VERSION=$(uname -r) && apt-cache show "linux-headers-${KERNEL_VERSION}" 2> /dev/null | sed -nE 's/^Version:\s+(([0-9]+\.){2}[0-9]+)[-.]([0-9]+).*/\1-\3/p' | head -1
-    ```
+1.  On Centos 7, we recommend using the latest kernel version in production, while the default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on centos, please see the following sections for detailed configuration. Please note that Ubuntu does not have the kernel version restriction.
```
Comment (samvantran) [inline pages/ksphere/konvoy/1.5.0-beta/gpu/index.md:36]:
```suggestion
1.  On Centos 7, we recommend using the latest kernel version in production, while the default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on Centos, please see the following sections for detailed configuration. Please note that Ubuntu does not have the kernel version restriction.
```

File: pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md [hunk 2]
```diff
@@ -15,6 +15,40 @@ enterprise: false
...
+
+- CLI: Only create a new `cluster.yaml` file if `.bkp` state file does not already exist.
+- CLI: Fix an error during `konvoy down` if there are non string terraform output variables defined.
+- Ansible: Fix Containerd upgrade from a cluster installed with a previous version of Konvoy.
+- Ansible: Fix an error where using `v1beta1` API in the `cluster.yaml` may cause Ansible to run certain tasks.
+- Ansible: Fix an error in the preflight checks that prevented installing on Ubuntu 18 (bionic).
+
+#### Improvements
+
+- CLI: Update Calico version to address the [CVE](https://docs.projectcalico.org/archive/v3.13/release-notes/&#35;v3134).
```
Comment (samvantran) [inline pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md:37]: This link points to a 404

File: pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md [hunk 3]
```diff
@@ -15,6 +15,40 @@ enterprise: false
...
+|**Maximum** | 1.17.x |
+|**Default** | 1.17.6 |
+
+#### Bug fixes
+
+- CLI: Only create a new `cluster.yaml` file if `.bkp` state file does not already exist.
+- CLI: Fix an error during `konvoy down` if there are non string terraform output variables defined.
+- Ansible: Fix Containerd upgrade from a cluster installed with a previous version of Konvoy.
+- Ansible: Fix an error where using `v1beta1` API in the `cluster.yaml` may cause Ansible to run certain tasks.
+- Ansible: Fix an error in the preflight checks that prevented installing on Ubuntu 18 (bionic).
```
Comment (dkoshkin) [inline pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md:32]:
```suggestion
- Ansible: Fix an error in the preflight checks that prevented installing on Ubuntu 18 (bionic).
- Ansible: Fix a bug where the GPU functionality did not work when using Containerd 1.3.4.
```

File: pages/ksphere/konvoy/1.5.0-beta/autoscaling/index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Konvoy uses the [cluster-autoscaler][autoscaler] community tool for autoscaling.
 
 <p class="message--note"><strong>NOTE: </strong> This feature is only supported for AWS and Azure cloud providers.</p>
 
-<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you **MUST** use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
+<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you **HAVE** to use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of this credentials
```
Comment (dkoshkin) [inline pages/ksphere/konvoy/1.5.0-beta/autoscaling/index.md:22]:
```suggestion
<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you <strong>HAVE</strong> to use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
```

File: pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md [hunk 5]
```diff
@@ -15,6 +15,41 @@ enterprise: false
...
+- Ansible: Fix a bug where the GPU functionality did not work when using Containerd 1.3.4.
+
+#### Improvements
+
+- CLI: Update Calico version to address the [CVE](https://docs.projectcalico.org/archive/v3.13/release-notes/&#35;v3134).
```
Comment (dkoshkin) [inline pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md:37]:
```suggestion
- CLI: Update Calico version to address the [CVE](https://docs.projectcalico.org/archive/v3.13/release-notes/#v3134).
```

File: pages/ksphere/konvoy/1.5.0-beta/autoscaling/index.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ Konvoy uses the [cluster-autoscaler][autoscaler] community tool for autoscaling.
 
 <p class="message--note"><strong>NOTE: </strong> This feature is only supported for AWS and Azure cloud providers.</p>
 
-<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you **MUST** use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
+<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you <strong>HAVE</strong> to use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/autoscaling/index.md:22]:
```suggestion
<p class="message--note"><strong>IMPORTANT: </strong> When using this feature, you <strong>must</strong> use cloud provider credentials with a longer expiration than the cluster's life or static credentials. The autoscaler makes usage of these credentials
```

File: pages/ksphere/konvoy/1.5.0-beta/gpu/index.md [hunk 7]
```diff
@@ -33,13 +33,7 @@ The following components provide Nvidia GPU support on Konvoy:
...
 1.  NVIDIA GPU with Fermie architecture version 2.1 or greater.
 
-1.  Ensure the kernel-headers version is identical to the kernel version on each node. To verify this use the following script:
-
-    * Ubuntu
-
-    ```bash
-    KERNEL_VERSION=$(uname -r) && apt-cache show "linux-headers-${KERNEL_VERSION}" 2> /dev/null | sed -nE 's/^Version:\s+(([0-9]+\.){2}[0-9]+)[-.]([0-9]+).*/\1-\3/p' | head -1
-    ```
+1.  On Centos 7, we recommend using the latest kernel version in production, while the default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on Centos, please see the following sections for detailed configuration. Please note that Ubuntu does not have the kernel version restriction.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/gpu/index.md:36]:
```suggestion
1.  On Centos 7, we recommend using the latest kernel version in production. The default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on Centos, refer to the following sections for detailed configuration. Note that Ubuntu does not have the kernel version restriction.
```

File: pages/ksphere/konvoy/1.5.0-beta/gpu/index.md [hunk 8]
```diff
@@ -79,6 +73,64 @@ spec:
       enabled: true
 ```
 
+### Nvidia Driver Configuration Based on Kernel Version
+
+Additional configuration may be needed if your kernel version of the GPU nodes is not identical to the latest one from the yum repository. In this case, you need to pick the right image tag for your Nvidia Driver container. For example, the following list is the supported kernel versions from the Nvidia upstream `nvidia/driver` image:
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/gpu/index.md:78]:
```suggestion
Additional configuration may be needed if your kernel version of the GPU nodes is not identical to the latest one from the yum repository. You need to pick the right image tag for your Nvidia Driver container. For example, the following list is the supported kernel versions from the Nvidia upstream `nvidia/driver` image:
```

File: pages/ksphere/konvoy/1.5.0-beta/gpu/index.md [hunk 9]
```diff
@@ -79,6 +73,64 @@ spec:
...
+418.40.04-3.10.0-957.10.1.el7.x86_64-centos7
+396.37-3.10.0-957.5.1.el7.x86_64-centos7
+396.37-3.10.0-957.1.3.el7.x86_64-centos7
+396.37-3.10.0-862.14.4.el7.x86_64-centos7
+396.37-3.10.0-862.14.4-centos7
+396.37-3.10.0-862.11.6-centos7
+396.37-3.10.0-862.9.1-centos7
+```
+
+This list is from [Nvidia Public Hub Repository][nvidia_public_hub_repository]. You would need to identify your kernel version first. For example,
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/gpu/index.md:104]:
```suggestion
This list is from [Nvidia Public Hub Repository][nvidia_public_hub_repository]. You need to identify your kernel version first. For example,
```

File: pages/ksphere/konvoy/1.5.0-beta/gpu/index.md [hunk 10]
```diff
@@ -79,6 +73,64 @@ spec:
...
+      enabled: true
+      values: |
+        nvidia-driver:
+          image:
+            tag: "440.33.01-3.10.0-1062.12.1.el7.x86_64-centos7"
+```
+
+### GPU on Air-gapped On-prem Cluster
+
+Please follow the [Konvoy Air-gapped Installations](../install/install-airgapped/) doc. Re-tag the `nvidia/driver` image with the corresponding tag that is identified from the above section, and push it to your local registry. For example:
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/gpu/index.md:124]:
```suggestion
Follow the [Konvoy Air-gapped Installations](../install/install-airgapped/) doc. Re-tag the `nvidia/driver` image with the corresponding tag, identified from the above section, and push it to your local registry. For example:
```

File: pages/ksphere/konvoy/1.5.0-beta/install/install-onprem/index.md [hunk 11]
```diff
@@ -37,7 +37,7 @@ Before installing, verify that your environment meets the following basic requir
   * 32 GiB memory
   * 80 GiB of free space in the root partition and the root partition must be less than 85% full.
 
-* If you plan to use **local volume provisioning** to provide [persistent volumes][persistent_volume] for the workloads, you must mount at least three volumes to `/mnt/disks/` mount point on each node. Each volume must have **at least** 55 GiB of capacity if the default addon configurations are used.  
+* If you plan to use **local volume provisioning** to provide [persistent volumes][persistent_volume] for the workloads, you must mount at least three volumes to `/mnt/disks/` mount point on each node. Each volume must have **at least** 55 GiB of capacity if the default addon configurations are used.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/install/install-onprem/index.md:40]:
```suggestion
* If you plan to use **local volume provisioning** to provide [persistent volumes][persistent_volume] for the workloads, you must mount at least three volumes to the `/mnt/disks/` mount point on each node. Each volume must have **at least** 55 GiB of capacity if the default addon configurations are used.
```

File: pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md [hunk 12]
```diff
@@ -15,6 +15,41 @@ enterprise: false
...
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+|**Minimum** | 1.15.4 |
+|**Maximum** | 1.17.x |
+|**Default** | 1.17.6 |
+
+#### Bug fixes
+
+- CLI: Only create a new `cluster.yaml` file if `.bkp` state file does not already exist.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md:28]:
```suggestion
- CLI: Only create a new `cluster.yaml` file if `.bkp` state file does not exist.
```

File: pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md [hunk 13]
```diff
@@ -15,6 +15,41 @@ enterprise: false
...
+- CLI: Fix an error during `konvoy down` if there are non string terraform output variables defined.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md:29]:
```suggestion
- CLI: Fix an error during `konvoy down` if there are non-string, terraform output variables, defined.
```

File: pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md [hunk 14]
```diff
@@ -15,6 +15,41 @@ enterprise: false
...
+- Ansible: Fix Containerd upgrade from a cluster installed with a previous version of Konvoy.
+- Ansible: Fix an error where using `v1beta1` API in the `cluster.yaml` may cause Ansible to run certain tasks.
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md:31]:
```suggestion
- Ansible: Fix an error where using `v1beta1` API in the `cluster.yaml` can cause Ansible to run certain tasks.
```

File: pages/ksphere/konvoy/1.5.0-beta/upgrade/upgrade-kubernetes-addons/index.md [hunk 15]
```diff
@@ -150,9 +155,34 @@ STAGE [Deploying Enabled Addons]
...
 You can also separate the above process by first running `konvoy deploy kubernetes --upgrade` and then running `konvoy deploy addons`.
 
-The steps above outline the general process for performing an upgrade of Kubernetes and addons. For some releases, the process might differ slightly. The steps below outline the process when upgrading to specific versions of Konvoy.
+The steps above outline the general process for performing an upgrade of Kubernetes and addons. For some releases, the process might differ slightly. See below for [actions required to upgrade from specific Konvoy versions](#actions-required-to-upgrade-from-specific-konvoy-versions).
+
+## Upgrade specific node pools
+
+You can have more control over the upgrade process by upgrading specific node pools. For example, you can upgrade only your control plane node pool, or only the control plane node pool and a single worker node pool.
+
+<p class="message--note"><strong>NOTE: </strong>Node pools allow the cluster administrator to use different configurations for different sets of worker nodes in a heterogeneous environment. For more, see the [Node Pools](../../install/node-pools/) page.</p>
```
Comment (russurquhart1) [inline pages/ksphere/konvoy/1.5.0-beta/upgrade/upgrade-kubernetes-addons/index.md:164]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Node pools allow the cluster administrator to use different configurations for different sets of worker nodes in a heterogeneous environment. For more, see the <a href="../../install/node-pools/">Node Pools</a> page.</p>

```

**Pull request merged**